server/db.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
can you move it into practice-one folder?

Reply:
I think my server.js wrote in commonjs type so I prefer to move out from practice-one folder @huyenpham-agilityio